packages/xo-vmdk-to-vhd/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "xo-vmdk-to-vhd",
-  "version": "0.1.6",
+  "version": "0.1.7",
```

Comment:
Don't change version numbers except in the changelog, it's done during the release process.